recipes/default.rb

```diff
@@ -17,7 +17,7 @@
 # limitations under the License.
 #
 package node['rsyslog']['package_name']
-package rsyslog_relp_package if node['rsyslog']['use_relp']
+package rsyslog_relp_package if node['rsyslog']['use_relp'] && unless platform_family?('freebsd')
```

Comment:
Apparently I need to drink more coffee to get this right. Apologies for not actually testing it on my end before suggesting. _This_ should work now.
```suggestion
package rsyslog_relp_package if node['rsyslog']['use_relp'] && !platform_family?('freebsd')
```